src/py/flwr/cli/new/templates/app/code/flwr_tune/strategy.py.tpl

```diff
@@ -0,0 +1,79 @@
+"""$project_name: A Flower / FlowerTune app."""
+
+from io import BytesIO
+from logging import INFO, WARN
+from typing import List, Tuple, Union
+
+from flwr.common import FitIns, FitRes, Parameters, log, parameters_to_ndarrays
+from flwr.server.client_manager import ClientManager
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.strategy import FedAvg
+
+
+class FlowerTuneLlm(FedAvg):
+    """Customised FedAvg strategy implementation."""
```

Comment:
```suggestion
    """Customised FedAvg strategy implementation.
    
    This class behaves just like FedAvg but also tracks the communication
    costs associated with a round of `fit`.
    """
```

Reply:
I thought of extending the description. Probably it can be done better than what i suggest above.